java/src/main/java/ola/OlaClient.java

```diff
@@ -180,6 +180,18 @@ public UniverseInfoReply getUniverseInfo(int universe) {
 
 
     /**
+     * Get list of all universe infos.
```

Comment:
This would probably be better written as "Get a list of info about all universes"